src/style.css

```diff
@@ -0,0 +1,141 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

Comment:
You need 2 different font-faces to plug fonts to your web-page

Reply:
Or do it like that `src: url(fonts/Avenir-Book.ttf), url(fonts/Avenir-Heavy.ttf);`